scripts/us_epa/superfund/sites/tar_creek/process_report2020.py

```diff
@@ -0,0 +1,208 @@
+
+    input_path = "./data/tar_creek_reports/2020_TarCreek.pdf"  #"https://semspub.epa.gov/work/06/100021610.pdf"
+    page_range = '83-86'
+    df_list = get_table_data_from_pdf(input_path, page_range)
+    columns = [
+        'observationAbout', 'observationDate', 'contaminantType', 'Cond.',
+        'D.O.', 'Hardness', 'temp', 'pH', 'Iron', 'Lead', 'Zinc', 'Cadmium',
+        'Sulfate'
+    ]
+    cleaned_dataset = [pd.DataFrame(columns=columns)]
+
+    for idx in range(len(df_list)):
+        df = df_list[idx].iloc[skip_count:]  #skip the first k-rows
+        extracted_well_name = df.iloc[0][0]
+        df = df[1:]  #skip the well name row
+        if idx == 3:
+            df['Unnamed: 0'] = df['Unnamed: 0'].replace(
+                {
+                    "/ ": "/",
+                    " /": "/",
+                    r'/ [^0-9]+': r'/[^0-9]+',
+                    'Tota ls': 'Totals',
+                    'Di sso lved': 'Dissolved',
+                    '200A': '2004',
+                    '19.E': '19.8'
+                },
+                regex=True)
+            df['Unnamed: 0'] = df['Unnamed: 0'].replace(
+                {'Dissolved': 'NaN Dissolved'})
+            df['Zinc'] = df['Zinc'].replace({'O.Ql': '0.01'})
+            df['Iron'] = df['Iron'].replace({
+                'O.Q': '0.0',
+                'O.o': '0.0'
+            },
+                                            regex=True)
+
+            df[['observationDate',
+                'contaminantType']] = df['Unnamed: 0'].str.split(n=1,
+                                                                 expand=True)
+            df['observationDate'] = pd.to_datetime(
+                df['observationDate']).ffill()
+            df['Temo. DH'] = df['Temo. DH'].replace({'19.E': '19.8'},
+                                                    regex=True)
+            df[['temp', 'pH']] = df['Temo. DH'].str.split(n=1, expand=True)
+            df['Hardness Cadmium'].replace(
+                to_replace=r'^<', value='- <', regex=True, inplace=True
+            )  #replace the entires with start with '<' since the after splitting the column associated with the data is wrong.
+            df[['Hardness',
+                'Cadmium']] = df['Hardness Cadmium'].str.split(n=1, expand=True)
+            df.drop(columns=[
+                'Unnamed: 0', 'Unnamed: 1', 'Temo. DH', 'Hardness Cadmium'
+            ],
+                    inplace=True)
+        if idx == 2:
+            df['Unnamed: 0'] = df['Unnamed: 0'].replace(
+                {
+                    "/ ": "/",
+                    " /": "/",
+                    r'/ [^0-9]+': r'/[^0-9]+',
+                    'To ta ls': 'Totals',
+                    'Total s': 'Totals',
+                    'Tota ls': 'Totals',
+                    'Tot als': 'Totals',
+                    'Di ssolved': 'Dissolved',
+                    'Di sso lved': 'Dissolved',
+                    '200A': '2004',
+                    '19.E': '19.8'
```

Comment:
move this out to a constant instead of repeating it for each index?

Reply:
Moved this to a constant